include/seahorn/seahorn.h

```diff
@@ -22,7 +22,7 @@ extern void __VERIFIER_error(void);
 extern void __VERIFIER_assume(int);
 extern void __SEA_assume(bool);
 
-extern void __VERIFIER_assert(bool);
+extern void __VERIFIER_assert(bool, int property);
```

Comment:
I think is is not backward compatible with existing clients of assert. 

You need to add a new function with new name. Not just change an existing one. 

Also this is probably the wrong place to add it unless you also change seahorn to recognize and treat it in some way.